core/templates/services/question-validation.service.ts

```diff
@@ -94,6 +122,17 @@ export class QuestionValidationService {
     if (!atLeastOneAnswerCorrect) {
       return 'At least one answer should be marked correct';
     }
+    if (
+      question._stateData.interaction.answerGroups.length > 1 ||
+      question._stateData.interaction.answerGroups[0].rules.length > 1
```

Comment:
What if there are no answer groups? Any test for this case?

Reply:
I think to use an interaction, we need to save the initial answer group when selecting it.

Reply:
Are you sure? I just went into a card and deleted all the answer groups.